proto/iamanager/v5/iamanager.proto

```diff
@@ -7,7 +7,7 @@ import "common/v1/common.proto";
 
 service IAMPublicService {
     rpc GetNodeInfo(google.protobuf.Empty) returns (NodeInfo) {}
-    rpc GetCert(GetCertRequest) returns (GetCertResponse) {}
+    rpc SubscribeCertChanged(SubscribeCertChangedRequest) returns (stream CertInfo) {}
```

Comment:
`SubscribeCertChanged` -> `SubscribeCert` as we receive not only change certificates.
`SubscribeCertChangedRequest` -> `SubscribeCertRequest`